src/main/kotlin/org/rust/ide/inspections/lints/RsUnknownCrateTypesInspection.kt

```diff
@@ -21,21 +19,28 @@ class RsUnknownCrateTypesInspection : RsLintInspection() {
 
     override fun buildVisitor(holder: RsProblemsHolder, isOnTheFly: Boolean): RsVisitor =
         object : RsVisitor() {
-            override fun visitLitExpr(o: RsLitExpr) {
-                val parent = o.parent as? RsMetaItem ?: return
+            override fun visitLitExpr(element: RsLitExpr) {
+                val parent = element.parent as? RsMetaItem ?: return
                 if (parent.path?.text != "crate_type") return
 
                 val context = ProcessingContext()
                 if (!parent.isRootMetaItem(context)) return
                 if (context.get(RsPsiPattern.META_ITEM_ATTR) !is RsInnerAttr) return
 
-                if (o.stringValue !in KNOWN_CRATE_TYPES) {
-                    holder.registerLintProblem(o, "Invalid `crate_type` value")
+                val elementValue = element.stringValue ?: return
+                if (elementValue !in KNOWN_CRATE_TYPES) {
+                    val fixes = NameSuggestionFix.createApplicable(
+                        element, elementValue, KNOWN_CRATE_TYPES, 1
+                    ) { name ->
+                        RsPsiFactory(element.project).createExpression("\"$name\"")
+                    }.toTypedArray()
```

Comment:
nit: Personally, I'd prefer the following variant

```suggestion
                    val fixes = NameSuggestionFix.createApplicable(element, elementValue, KNOWN_CRATE_TYPES, 1) {
                        RsPsiFactory(element.project).createExpression("\"$it\"")
                    }.toTypedArray()
```

but it's up to you